sdk/rust/sdk/src/chains/terra.rs

```diff
@@ -33,19 +33,16 @@ pub fn id() -> Addr {
 /// Export Core Devnet Contract Address
 #[cfg(feature = "devnet")]
 pub fn id() -> Addr {
-    Addr::unchecked("terra1pd65m0q9tl3v8znnz5f5ltsfegyzah7g42cx5v")
+    Addr::unchecked("terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5")
```

Comment:
This is not the devnet address according to the docs

Reply:
`terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5` seems to be correct, the previous address here was testnet not devnet so this looks like the right address to me now.